content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,33 @@
+**Title**
+
+**HTML global attribute**
```

Comment:
Do not use title headers before the initial introduction to the term. The metadata will provide that.